_config/queuedjobs.yml

```diff
@@ -7,6 +7,7 @@ Injector:
   QueuedJobService:
     properties: 
       queueHandler: %$QueueHandler
+      queueRunner: %$DoormanRunner
```

Comment:
Or you could just change the getService method in BaseRunner to `return singleton('QueuedJobService')`